vayesta/solver/coupling.py

```diff
@@ -280,52 +283,181 @@ def _integrals_for_extcorr(fragment, fock):
         govovbb = eris[ob,vb,ob,vb]
```

Comment:
I think I wrote this, but this probably needs to be `eris[0]`, `eris[1]`, and `eris[2]` - not that UHF is working anyways

Reply:
Made a comment in code to address this when it comes to implementing UHF